src/scss/cozy.scss

```diff
@@ -126,4 +126,10 @@ app-vault-ciphers {
             } 
         }
     }
+}
+
+#login-page {
+    .error-text {
+        color: var(--secondaryColorLightest);
```

Comment:
We can't use errorColor or something more semantic than this one? It seems a bit weird.

Reply:
This is because Inverted CSS is not loaded in the project, so I only have access to normal Theme semantic names. Current config include `cozy-ui/transpiled/react/stylesheet.css`, but this file does not contain the inverted css variables.

On normal themes, errors have those colors:
- errorColor: `#f52d2d`
- errorColorDark: `#a00808`
- errorColorLight: `#fcbfbf` <-- maybe the closest
- errorColorLightest: `#fee6e6`

On inverted theme and with new cozy-ui version (with themes rework), those are:
- errorColorLight: `#ffecec`
- errorColor: `#ffd3d3` <- the one I want
- errorColorDark: `#ffb9b9`

I'm not sure how to include inverted theme in an angular project that doesn't use stylus.

So my guess is that we have the following choices:
- find how to import inverted vars (I would appreciate some hints on how to do it)
- use `--secondaryColorLightest` that is not semantic but has the correct value (depending on which cozy-ui version we use)
- use `--errorColorLight` from normal theme, that is semantically inexact (but very close) and doesn't have the correct color (but very close too)

Reply:
> . Current config include cozy-ui/transpiled/react/stylesheet.css, but this file does not contain the inverted css variables.

1/ This file is for "react components". https://github.com/cozy/cozy-ui#as-a-components-library 
2/ If you use don't use React, you can import UI as vanilla css https://github.com/cozy/cozy-ui#as-a-vanilla-css-library and there it seems that inverted theme is exported. 

We will not support 2 in UI in the future. 

1/ Should work, it seems that https://unpkg.com/cozy-ui@54.0.0/transpiled/react/stylesheet.css inverted-theme is present in the generated file. But I think that cozy-pass doesn't support css module so this is why when you use the react stylesheet and write "CozyTheme--inverted" we don't get anything since this is not converted to `.styles__CozyTheme--inverted___1ygHa`. right?

Reply:
> If you use don't use React, you can import UI as vanilla css

Woups, I copied the wrong line, but both `cozy-ui.min.css` and `react/stylesheet.css` are imported : https://github.com/cozy/cozy-pass-web/blob/1c5c0eb8f7d750ca30818993b33061892f35c0e2/src/scss/cozy.scss#L1-L4

`react/stylesheet.css` is used for imported React components.

> We will not support 2 in UI in the future.

Do you have more details about this?

> 1/ Should work, it seems that https://unpkg.com/cozy-ui@54.0.0/transpiled/react/stylesheet.css inverted-theme is present in the generated file. But I think that cozy-pass doesn't support css module so this is why when you use the react stylesheet and write "CozyTheme--inverted" we don't get anything since this is not converted to .styles__CozyTheme--inverted___1ygHa. right?

Right, and I don't want to use `styles__CozyTheme--inverted___1ygHa` class name as it may change on next builds?

Reply:
> Right, and I don't want to use styles__CozyTheme--inverted___1ygHa class name as it may change on next builds?

Yep. We need to add support for css module on this web app. We've some webpack files for that on cozy-scripts. 

>>We will not support 2 in UI in the future.
>Do you have more details about this?

Yep. Cozy-UI will be only for React Application. We've dropped the support for "vanilla css". Instead we've a new project called cozy-bootstrap for that ;)